app/controllers/staff/reports_submissions_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Staff::ReportsSubmissionsController < Staff::BaseController
+  include Secured
+
+  def show
+    report = Report.find(params[:report_id])
+    authorize report
+    @report_presenter = ReportPresenter.new(report)
+    render "staff/reports/submit"
+  end
+
+  def update
+    @report = Report.find(params[:report_id])
+    authorize @report
+
+    @report.state = :submitted
```

Comment:
This isn't necessary right now, but as we expand this to handle more states, we should check that the requested state _transition_ is valid -- you can't mark a report `submitted` if it's currently `inactive`, for example. There are gems that can help with this, for example [aasm](https://github.com/aasm/aasm).